man/CMakeLists.txt

```diff
@@ -1,43 +1,60 @@
-install(FILES
-	ceph-osd.8
-	ceph-mds.8
-	ceph-mon.8
-	ceph-syn.8
-	crushtool.8
-	osdmaptool.8
-	monmaptool.8
-	ceph-conf.8
-	ceph-run.8
-	ceph.8
-	mount.ceph.8
-	ceph-create-keys.8
-	radosgw.8
-	radosgw-admin.8
-	ceph-authtool.8
-	rados.8
-	librados-config.8
-	ceph-clsinfo.8
-	ceph-debugpack.8
-	cephfs.8
-	ceph-dencoder.8
-	ceph-rest-api.8
-	ceph-post-file.8
-  DESTINATION ${CEPH_MAN_DIR}/man8)
+set(manpages_srcs
+  8/ceph-osd.rst
+  8/ceph-mds.rst
+  8/ceph-mon.rst
+  8/ceph-syn.rst
+  8/crushtool.rst
+  8/osdmaptool.rst
+  8/monmaptool.rst
+  8/ceph-conf.rst
+  8/ceph-run.rst
+  8/ceph.rst
+  8/mount.ceph.rst
+  8/ceph-create-keys.rst
+  8/radosgw.rst
+  8/radosgw-admin.rst
+  8/ceph-authtool.rst
+  8/rados.rst
+  8/librados-config.rst
+  8/ceph-clsinfo.rst
+  8/ceph-debugpack.rst
+  8/cephfs.rst
+  8/ceph-dencoder.rst
+  8/ceph-rest-api.rst
+  8/ceph-post-file.rst)
 
 if(HAVE_LIBFUSE)
-  install(FILES
-	ceph-fuse.8
-	rbd-fuse.8
-	DESTINATION ${CEPH_MAN_DIR}/man8)
+  list(APPEND manpage_srcs
+    8/ceph-fuse.rst
+    8/rbd-fuse.rst)
 endif()
 
 if(WITH_RBD)
-  install(FILES
-	ceph-rbdnamer.8
-	rbd-nbd.8
-	rbd-replay-prep.8
-	rbd-replay.8
-	rbdmap.8
-	rbd.8
-	DESTINATION ${CEPH_MAN_DIR}/man8)
+  list(APPEND manpage_srcs
+	8/ceph-rbdnamer.rst
+	8/rbd-nbd.rst
+	8/rbd-replay-prep.rst
+	8/rbd-replay.rst
+	8/rbdmap.rst
+	8/rbd.rst)
 endif()
+
+foreach(manpage ${manpage_srcs})
+  string(REGEX MATCH "^[0-9]/" section ${manpage})
+  get_filename_component(cmd ${manpage} NAME_WE)
+  list(APPEND sphinx_input ${CMAKE_SOURCE_DIR}/doc/man/${manpage})
+  list(APPEND sphinx_output ${CMAKE_BINARY_DIR}/doc/man/${cmd}.${section})
+endforeach()
+
+install(FILES ${sphinx_output} ceph_selinux.8
+  DESTINATION ${CEPH_MAN_DIR}/man8)
```

Comment:
we could use more sophisticated way to determine the section instead hard-wiring it to man8. but since we only *.8 man pages at this moment, let install all of them into "man8" now.